### ISSUE TITLE
Quorum Queues : No new leader is be selected if primary node goes down

### ISSUE THREAD

**Issue by k123009** [rabbitmq/rabbitmq-server#2390]
I am doing POC on RabbitMQ's Quorum Queues, especially focusing on fail-over mechanism. In my case I have two nodes (for example NodeA and NodeB) and one Quorum queue which resides on NodeA. Now whenever I am publishing a test message to Quorum queue of NodeA, I can see the same message on NodeB.

Now when testing the failover mechanism and stopping NodeA, I am unable to publish any message, also I can not see any messages in quorum queue, I think the NodeB is not promoted to be a new leader. I am supposing the leader would be promoted automatically, do I need to do anything to make the other Node leader ?

**Comment by michaelklishin** [comment]
Thank you for your time.

Team RabbitMQ uses GitHub issues for specific actionable items engineers can work on. GitHub issues are not used for questions, investigations, root cause analysis, discussions of potential issues, etc (as defined by this team).

We get at least a dozen of questions through various venues every single day, often light on details.
At that rate GitHub issues can very quickly turn into a something impossible to navigate and make sense of even for our team. Because GitHub is a tool our team uses heavily nearly every day, the signal/noise ratio of issues is something we care about a lot.

Please post this to [rabbitmq-users](https://groups.google.com/forum/#!forum/rabbitmq-users).

Thank you.

**Comment by michaelklishin** [comment]
TL;DR: quorum queues do not support two nodes clusters and two-node clusters are [highly recommended against](https://www.rabbitmq.com/clustering.html#node-count) for any clusters.

From Quorum Queues documentation guide:

```
A quorum queue requires a quorum of the declared nodes to be available to function.
When a RabbitMQ node hosting a quorum queue's leader fails or is stopped another node
hosting one of that quorum queue's follower will be elected leader and resume operations.
```

([Leader Election and Failure Handling](https://www.rabbitmq.com/quorum-queues.html#leader-election))

```
Failure tolerance characteristics of clusters of various size can be described in a table…
```

and the table clearly states that a two node cluster will tolerate failure of 0 nodes.

([Fault Tolerance and Minimum Number of Replicas Online](https://www.rabbitmq.com/quorum-queues.html#quorum-requirements))
